modules/servers/pvewhmcs/pvewhmcs.php

```
@@ -1244,8 +1244,10 @@ function pvewhmcs_noVNC($params) {
 		$vncticket = $vm_vncproxy['ticket'];
 		// $path should only contain the actual path without any query parameters
 		$path = 'api2/json/nodes/' . $guest_node . '/' . $guest->vtype . '/' . $guest->vmid . '/vncwebsocket?port=' . $vm_vncproxy['port'] . '&vncticket=' . urlencode($vncticket);
+		// Get WHMCS base URL (including subdirectory
+		$whmcs_base = $CONFIG['SystemURL'];
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$CONFIG'. (undefined)

(UnusedLocalVariable)